modules/hellonik/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, pkgs, config, ... }:
+with lib;
+let cfg = config.mayniklas.services.hellonik;
+in {
+  options.pinpox.services.hellonik = {
```

Comment:
In line 3 wird `mayniklas.services.hellonik` verwendet.
In line 5 `pinpox.services.hellonik`.
Beide Bezeichnungen müssen miteinander einstimmen.
-> wenn beide gleich sind, dann klappt alles.